scripts/release_tag.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+TAG=${1:-0.3.1}
```

Comment:
We should update this script to update a version.
https://github.com/containers/youki/blob/9b03363d628b67cc762a2c5a6b02387d55d2b069/justfile#L192-L195

Reply:
How about exiting with failing if a user doesn't pass the $1?

Reply:
> How about exiting with failing if a user doesn't pass the $1?

Of course, the user must fill in this parameter.

Reply:
> We should update this script to update a version.
> 
> https://github.com/containers/youki/blob/9b03363d628b67cc762a2c5a6b02387d55d2b069/justfile#L192-L195

Okay, I will modify it together later.